engines/advancedDetector.cpp

```diff
@@ -513,31 +503,56 @@ namespace Common {
 	DECLARE_SINGLETON(MD5CacheManager);
 }
 
-// Sync with engines/game.cpp
-static char flagsToMD5Prefix(uint32 flags) {
-	if (flags & ADGF_MACRESFORK) {
-		if (flags & ADGF_TAILMD5)
-			return 'e';
-		return 'm';
+
+static MD5Properties gameFileToMD5Props(const ADGameFileDescription *fileEntry, uint32 gameFlags) {
+	if(fileEntry && fileEntry->md5 && fileEntry->md5[0] != '\0'  && fileEntry->md5[1] == ':') {
```

Comment:
Bad formatting. `if (` and there are two spaces after `'\0'`

Reply:
Done